src/command/identity/import.ts

```diff
@@ -18,10 +18,10 @@ export class Import extends RootCommand implements LeafCommand {
   @Argument({ key: 'path', required: true, describe: 'Path to the V3 wallet file' })
   public path!: string
 
-  @Option({ key: 'identityName', describe: 'Name of the identity to be saved as' })
+  @Option({ key: 'identityName', alias: 'i', describe: 'Name of the identity to be saved as' })
```

Comment:
in `export.ts` it is defines with dashes. let's keep that convention in key string names instead of this one